index.html

```diff
@@ -1,29 +1,96 @@
-            <input type="email" id="email">
-            <button class="btn submit">Daj nam znać!</button>
-        </form>
-    </div>
 
+<body>
+    <header class="header">
+        <div class="header__logo">Logo</div>
+        <nav class="nav header__nav">
+            <ul class="nav__list">
+                <li class="nav__item"><a href="https://codeforpoznan.pl/" class="nav__link">O nas</a></li>
+                <li class="nav__item"><a href="#" class="nav__link">Pomóż Alince</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="intro">
+        <h1 class="intro__title">Alinka</h1>
+        <h2 class="intro__description">tekst opisujący/zachęcający</h2>
+        <button class="button intro__button button--regular">Ściągnij</button>
+    </section>
+    <section class="description">
+        <h1 class="titles description__title">Automatyczne generowanie dokumentów</h1>
+        <p class="decription__text">Alinka jest aplikacją desktopową, która ma pomagać osobom z polskich Poradni
+            Psychologiczno-Pedagogicznym
+            odpowiedzialnym za wydawanie orzeczeń.</p>
+        <p class="decription__text">Idea powstała, dzięki Pani Alince wykonującą tę pracę. Wiąże się ona ze żmudnym
+            wypełnianiem tych samym
+            danych w wielu dokumentach, a tym samym prowadzi do godzin niepotrzebnej pracy. Chcąc wesprzeć cały
+            proces stworzyliśmy narzędzie, które zautomatyzuje generowanie dokumentów zgodnych z oficjalnymi
+            standardami Ministerstwa Edukacji Narodowej. Tym samym wzrośnie efektywność oraz zmaleje ilość błędów w
+            porównaniu z ręcznym wprowadzaniem danych.</p>
+        <p class="decription__text">Generowany dokument zostanie utworzony jako plik .docs, możliwy do edycji w Wordzie
+            i ewentualnego
+            dopisania dodatkowych informacji.</p>
+        <img src="assets/images/description-graphics.svg" alt="" class="description__graphics">
+    </section>
+    <section class="demo">
+        <h1 class="titles demo__title">Zobacz Alinkę w akcji</h1>
+        <div class="demo__box">demo</div>
+        <img src="assets/images/wave-left.svg" alt="" class="demo__wave--left">
+        <img src="assets/images/wave-right.svg" alt="" class="demo__wave--right">
+    </section>
+    <section class="features">
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-free.svg" alt="" class="feature__icon">
+            <p class="feature__text">Za darmo</p>
+        </dev>
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-easy.svg" alt="" class="feature__icon">
+            <p class="feature__text">Łatwa w osłudze</p>
+        </dev>
+        <dev class="feature features__feature">
+            <img src="assets/images/feature-safe.svg" alt="" class="feature__icon">
+            <p class="feature__text">Bezpieczna</p>
+        </dev>
+    </section>
```

Comment:
MDN documentation:
```
- Each <section> should be identified, typically by including a heading (<h1>-<h6> element) as a child of the <section> element.

- Do not use the <section> element as a generic container; this is what <div> is for, especially when the sectioning is only for styling purposes. A rule of thumb is that a section should logically appear in the outline of a document.
```

1.  maybe it would be better to include those features into the previous section and rename it (no class="demo") because this section it will be showing all the best of Alinka :D so demo as well as these features could be in one container

2. features__feature is not good idea of naming class, you could use "ul -> features__list" and then "li -> features__item"

3. we don't have tag `<dev>` :D